lighthouse-cli/chrome-launcher.ts

```diff
@@ -136,6 +136,11 @@ class ChromeLauncher {
 
   spawn(execPath: string): Promise<any[]> {
     return new Promise((resolve, reject) => {
+      if (this.chrome) {
+        log.verbose('ChromeLauncher', `Chrome already running with pid ${this.chrome.pid}.`);
```

Comment:
It's all the same for me, I just took whatever was underneath. Just let me know what you prefer